### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by kolbma** [tiny-http/tiny-http#231] [General]
Thank you @tomaka, @frewsxcv, @bradfier and all the other [contributors](https://github.com/tiny-http/tiny-http/graphs/contributors).

I'm happy to have found your project.  
It makes fun to use.  
It works like imagined and there is no type-hell with endless wrappers and incompatible traits.

Will also try out [Rouille](https://github.com/tomaka/rouille) when I need something more RESTy...

:partying_face: :joy: :+1: :1st_place_medal:

**Comment by ghost** [comment]
Hi everyone.

> I'm happy to have found your project.

I too.
